Shelfie.Identity/Shelfie.Identity.BusinessLogic/UseCases/RegisterUser/RegisterUserCommandHandler.cs

```diff
@@ -0,0 +1,33 @@
+﻿using MediatR;
+using Microsoft.AspNetCore.Identity;
+
+namespace Shelfie.Identity.BusinessLogic.UseCases.RegisterUser;
+
+public class RegisterUserCommandHandler : IRequestHandler<RegisterUserCommand, IdentityResult>
+{
+    private readonly UserManager<IdentityUser> _userManager;
+
+    public RegisterUserCommandHandler(UserManager<IdentityUser> userManager)
+    {
+        _userManager = userManager;
+    }
+
+    public async Task<IdentityResult> Handle(RegisterUserCommand request, CancellationToken cancellationToken)
```

Comment:
You shouldn't return identity models outside handler in this layer. Please your custom class.

Reply:
Consider creating your own custom OperationResult